certs_to_crlite.py

```diff
@@ -0,0 +1,49 @@
+import censys.certificates
+from decouple import config
+from filter_cascade import FilterCascade
+import sys
+
+CENSYS_API_UID = config("CENSYS_API_UID", None)
+CENSYS_API_SECRET = config("CENSYS_API_SECRET", None)
+
+if not CENSYS_API_UID or not CENSYS_API_SECRET:
+    print "Must set CENSYS_API_UID and CENSYS_API_SECRET"
+    sys.exit(1)
+
+certificates = censys.certificates.CensysCertificates(
+    CENSYS_API_UID, CENSYS_API_SECRET
+)
+
+MLBF_FILENAME = 'moz-crlite-mlbf'
+
+a = []
+b = []
+
+certs_list = certificates.search(
+    'parsed.issuer.organization.raw: "DigiCert Inc"'
+)
+
+for idx, c in enumerate(certs_list):
+    print "Iteration %i" % idx
+    if idx == 500:
+        break
+    # let's say every third cert should *not* be in the filter
+    shouldBeRevoked = (idx % 3 == 1)
+    if shouldBeRevoked:
+        print "Appending %s to revoked list." % c
+        a.append(c["parsed.fingerprint_sha256"])
+    else:
+        print "Appending %s to okay list." % c
+        b.append(c["parsed.fingerprint_sha256"])
+
+cascade = FilterCascade(500, 1.3, 0.77, 1)
```

Comment:
I seem to recall there's a section of the paper or implementer's guide that covers the choice of values for capacity and error rate. I'll refresh my memory on this. I think the above values (from my example) were based on measurements from the word set I experimented with.

Reply:
Yup ...

> To calculate the parameters for the filter cascade, two variables are required:
r – the maximum number of revoked certificates to support in this filter cascade
s – the maximum number of non-revoked certificates to support

> We fix r = 1,500,000 and s = 70,000,000, which yields a constant uncompressed filter cascade size of 2.8 MB.